projects/plugins/boost/app/assets/src/css/components/critical-css.scss

```diff
@@ -90,10 +90,14 @@
 	}
 
 	.summary {
-		margin-right: 5px;
+		margin-right: 120px;
 		flex-grow: 1;
 		position: relative;
 
+		@media (max-width: 1080px) {
+			margin-right: 5px;
+		}
```

Comment:
🪄 Magic number detected.

Should this be a `@include breakpoint()` or is there some specific edge case for 1080px (in which case a comment would be good to have here).

Reply:
Reverted with https://github.com/Automattic/jetpack/pull/29250/commits/80fbcfb4b624a679f8fb2ebfef43f664b01f0094 (ignore the commit name 😅).